src/lib/classNames.test.ts

```diff
@@ -8,7 +8,7 @@ describe(classNames, () => {
     expect(
       classNames({ b: true, no: false, c: true, no2: null, no3: undefined })
     ).toEqual(["b", "c"]));
-  it("filters out falsey values", () =>
+  it("filters out false values", () =>
```

Comment:
```suggestion
  it("filters out falsy values", () =>
```

См. https://developer.mozilla.org/en-US/docs/Glossary/Falsy